data/campaigns/tutorial04_economy.wmf/scripting/texts.lua

```diff
@@ -303,6 +303,7 @@ building_priority_settings = {
       li_object("empire_marblemine", p(_[[Great. Our taverns have now been built up and are supplying us with rations.]]) ..
          p(_[[At the moment, all mines are supplied with rations. If you want to prioritize a special mine, you simply have to open its window. In the wares tab, behind every ware, you can see ‘traffic lights’.]]) ..
          p(_[[When you click on the red dot (low priority), the corresponding ware gets delivered less frequently. Green means that as many wares as possible should be delivered to this building, maybe because it produces something important.]]) ..
+         p(_[[The actual supply state is indicated by varying colors of the wares icons themselves: Wares stored in the building are shown in full color. Missing wares are greyed out. Wares not present in the building yet but already being transported there are shown in a darker grey.]]) ..
```

Comment:
wares icons -> ware icons

I'd also put the 2 cases into bullet points to break up the wall of text a bit.

Reply:
done